site2/website/release-notes.md

```diff
@@ -1,6 +1,438 @@
+
+### Pulsar IO and Pulsar Functions
+
+- Added Debezium source for Microsoft SQL Server. [12256](https://github.com/apache/pulsar/pull/12256)
+- Upgrading Debezium to 1.7. [12295](https://github.com/apache/pulsar/pull/12295)
+- Allow Pulsar Functions localrun to exit on error. [12278](https://github.com/apache/pulsar/pull/12278)
+- [pulsar-io-kinesis] add missing dependency. [12246](https://github.com/apache/pulsar/pull/12246)
+- Support for draining workers. [12178](https://github.com/apache/pulsar/pull/12178)
+- [pulsar-functions-go] support set subscription position. [11990](https://github.com/apache/pulsar/pull/11990)
+- [function] enable protobuf-native schema support for function. [11868](https://github.com/apache/pulsar/pull/11868)
+- [pulsar-functions-go] sync to the latest function proto. [11853](https://github.com/apache/pulsar/pull/11853)
+- [Functions]Support protobuf schema for pulsar function. [11709](https://github.com/apache/pulsar/pull/11709)
+- [Functions] Remove unused class which didn't get reverted. [11707](https://github.com/apache/pulsar/pull/11707)
+- [Functions] Support KEY_BASED batch builder for Java based functions and sources. [11706](https://github.com/apache/pulsar/pull/11706)
+- PIP-85: [pulsar-io] pass pulsar client via context to connector. [11056](https://github.com/apache/pulsar/pull/11056)
+
+### Java Client
+- [Java Client] Use failPendingMessages to ensure proper cleanup. [12259](https://github.com/apache/pulsar/pull/12259)
+- [Java Client] Remove unnecessary stats incremement. [12229](https://github.com/apache/pulsar/pull/12229)
+- [Java Client] Remove unnecessary synchronization and simplify initialization. [12228](https://github.com/apache/pulsar/pull/12228)
+- [pulsar-client] Add partition-change API for producer or consumer interceptors. [12287](https://github.com/apache/pulsar/pull/12287)
+- Fix deadLetterPolicy is not working with key shared subscription under partitioned topic. [12148](https://github.com/apache/pulsar/pull/12148)
+- [Client] Fix endless receiveAsync loop in MultiTopicsConsumer. [12044](https://github.com/apache/pulsar/pull/12044)
+- [Client] Reduce redundant FLOW requests for non-durable multi-topics consumer. [11802](https://github.com/apache/pulsar/pull/11802)
+- [pulsar-client] Set and return topic names on message API. [11743](https://github.com/apache/pulsar/pull/11743)
+- PIP-91: Separate lookup timeout from operation timeout. [11627](https://github.com/apache/pulsar/pull/11627)
+- [Java Client] Switch from pretty print to compact print for configs. [11609](https://github.com/apache/pulsar/pull/11609)
+- Java Client: remove usage of reflection while using Pulsar Implementation classes. [11636](https://github.com/apache/pulsar/pull/11636)
+- [pulsar-client]Add a optional params scope for pulsar oauth2 client. [11931](https://github.com/apache/pulsar/pull/11931)
```

Comment:
1. Since these items belong to `Java client`, can we delete [Java client] for each item to make content more concise?
![image](https://user-images.githubusercontent.com/50226895/144012351-79de642d-d1dc-4192-a8c9-2f3c9cd3de90.png)

2. Same comments for other sections

Reply:
done